library/Ivoz/Cgr/Infrastructure/Persistence/Doctrine/TpAccountActionDoctrineRepository.php

```diff
@@ -19,4 +19,16 @@ public function __construct(RegistryInterface $registry)
     {
         parent::__construct($registry, TpAccountAction::class);
     }
+
+    /**
+     * @param int $companyId
+     */
+    public function findByCompany(int $companyId)
+    {
+        $criteria = [
+            'company' => $companyId
+        ];
+
+        return $this->findOneBy($criteria);
+    }
```

Comment:
Is there a reason not to pass _CompanyInterface_ here and do `$company->getId()` in criteria?
Should this be called _findByCompanyId_?